server/db/migrate/20200426015709_add_fields_to_user_profile.rb

```diff
@@ -0,0 +1,10 @@
+class AddFieldsToUserProfile < ActiveRecord::Migration[6.0]
+  def change
+    add_column :user_profiles, :business_name, :string, :after => :user_name
+    add_column :user_profiles, :specialty, :string, :after => :user_name
+    add_column :user_profiles, :tip_url, :string, :after => :user_name
```

Comment:
We have a `donation_methods` table already for this. We did it as a separate table to allow users to be able to have multiple links, say to both a paypal and a venmo

Reply:
Yeah I saw that, and I tried to add a row to that table but was spending too much time trying to figure out RoR so I just added a field to the main profiles table for now. We can go back to the one-to-many relationship once I get a bit more ramped up on Rails and understand how to do it properly!